sova/dashboard/services/settings_service.py

```
@@ -47,35 +45,52 @@ def _flatten_dict(prefix: str, obj: dict, result: dict, registered: frozenset[st
+        toml_path.write_text(tomlkit.dumps(doc))
         return {"status": "ok", "key": key, "value": value}
-    except OperationalError:
-        log.warning("settings.update.db_not_initialized", key=key, exc_info=True)
-        return {"error": "Database not initialized. Run 'sova init-db' first."}
-    except Exception:
-        log.warning("settings.update.error", key=key, exc_info=True)
-        return {"error": "Failed to persist configuration"}
+    except ImportError:
+        log.warning("tomlkit not available, skipping config update")
+        return {"error": "tomlkit not installed -- cannot update config"}
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)